pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -501,13 +503,33 @@ func (c *Cluster) DeployPrometheusRule(name, namespace string) error {
 	if err != nil {
 		return errors.Wrapf(err, "failed to set owner reference to prometheus rule %q", prometheusRule.Name)
 	}
+
 	cephv1.GetMonitoringLabels(c.spec.Labels).ApplyToObjectMeta(&prometheusRule.ObjectMeta)
+	applyDefaultMonitoringLabels(&prometheusRule.ObjectMeta)
+
 	if _, err := k8sutil.CreateOrUpdatePrometheusRule(prometheusRule); err != nil {
 		return errors.Wrap(err, "prometheus rule could not be deployed")
 	}
 	return nil
 }
 
+func applyDefaultMonitoringLabels(m *metav1.ObjectMeta) {
+	defaultLabels := map[string]string{
+		"prometheus": "rook-prometheus",
+		"team":       "rook",
```

Comment:
this label was only on the `ServiceMonitor`s before, now it's also at `PrometheusRule`s. With this one I'm not sure if it is useful to have by default (as with the previous set `role` label on the `PrometheusRule`s).